src/main/java/org/levelup/labradoodle/repositories/DishRepository.java

```diff
@@ -0,0 +1,58 @@
+package org.levelup.labradoodle.repositories;
+
+import org.levelup.labradoodle.models.entities.Dishes;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import java.util.List;
+
+/**
+ * Created by RomanMosiienko on 18.08.15.
+ */
+@Repository
+@Transactional
```

Comment:
Comments just with author is not enough. I want all of you write comment for class and all public methods